user-registration-app/src/app/services/AuthService/auth.service.ts

```diff
@@ -27,33 +27,33 @@ export class AuthService {
   }
 
   signIn(email: string, password: string) {
-    return this.afAuth.auth.signInWithEmailAndPassword(email, password);
+    return this.afAuth.signInWithEmailAndPassword(email, password);
   }
 
   signInWithProvider(provider: AuthProviders): Promise<any> {
-    let authProvider: firebase.auth.AuthProvider = null;
+    let authProvider: app.auth.AuthProvider = null;
     switch (provider) {
       case AuthProviders.GOOGLE_PROVIDER:
-        authProvider = new firebase.auth.GoogleAuthProvider();
+        authProvider = new app.auth.GoogleAuthProvider();
         break;
       case AuthProviders.GITHUB_PROVIDER:
-        authProvider = new firebase.auth.GithubAuthProvider();
+        authProvider = new app.auth.GithubAuthProvider();
         break;
       case AuthProviders.APPLE_PROVIDER:
-        const provider = new firebase.auth.OAuthProvider('apple.com');
-        provider.addScope('email');
-        provider.addScope('name');
-        authProvider = provider;
+        const oAuthProvider = new app.auth.OAuthProvider('apple.com');
+        oAuthProvider.addScope('email');
+        oAuthProvider.addScope('name');
+        authProvider = oAuthProvider;
```

Comment:
The linter didn't like that the new variable overshadowed the function parameter